ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -1216,13 +1227,14 @@ void VCCueList::slotSideFaderValueChanged(int value)
         int newStep = value; // by default we assume the Chaser has more than 256 steps
         if (ch->stepsCount() < 256)
         {
-            float stepSize = 255.0 / (float)ch->stepsCount();
-            if (value >= 255.0 - stepSize)
+            float stepSize = 256.0 / (float)ch->stepsCount();  //divide up the full 0..255 range
+            stepSize = (float)qFloor((stepSize * 100000) + 0.5) / 100000; //round to 5 decimals to fix corner cases
```

Comment:
Cast not needed. Explicitly use 100000.0

Reply:
Tested this as per your recommendation and it works fine that way.  Casts removed.